src/style/theme.js

```diff
@@ -0,0 +1,105 @@
+import { createTheme } from '@mui/material/styles';
+
+export const theme = createTheme({
+    palette: {
+        type: 'light',
+        primary: {
+            main: '#00FFD1',
+        },
+        secondary: {
+            main: '#ff79b0',
+        },
+    },
+    typography: {
+        fontFamily: 'Ubuntu Mono',
+        button: {
+            fontSize: '1.15rem',
+            textTransform: 'none',
+            fontWeight: 600,
+        },
+    },
+    spacing: 8,
+    shape: {
+        borderRadius: 4,
+    },
+    components: {
+        MuiTooltip: {
+            styleOverrides: {
+                tooltip: {
+                    padding: '10px',
+                    fontSize: '0.75rem',
+                },
+            }
+        },
+        MuiDialog: {
+            styleOverrides: {
+                paper: {
+                    backgroundColor: '#444444'
+                }
+            }
+        },
+        MuiCssBaseline: {
+            styleOverrides: {
+                body: {
+                    "&::-webkit-scrollbar, & *::-webkit-scrollbar": {
+                        "-webkit-appearance": "none",
+                        width: 10,
+                    },
+                    "&::-webkit-scrollbar-track, & *::-webkit-scrollbar-track": {
+                        backgroundColor: "rgba(0, 0, 0, 0.1)",
+                    },
+                    "&::-webkit-scrollbar-thumb, & *::-webkit-scrollbar-thumb": {
+                        borderRadius: 8,
+                        backgroundColor: "rgba(0, 0, 0, 0.25)",
+                    },
+                    "&::-webkit-scrollbar-thumb:focus, & *::-webkit-scrollbar-thumb:focus": {
+                        backgroundColor: "rgba(0, 0, 0, 0.25)",
+                    },
+                    "&::-webkit-scrollbar-thumb:hover, & *::-webkit-scrollbar-thumb:hover": {
+                        backgroundColor: "rgba(128, 135, 139, 0.8)",
+                    },
+                }
+            }
+        }
+    },
+    props: {
+        MuiButton: {
+            size: 'small',
+        },
+        MuiButtonGroup: {
+            size: 'small',
+        },
+        MuiCheckbox: {
+            size: 'small',
+        },
+        MuiFab: {
+            size: 'small',
+        },
+        MuiFormControl: {
+            margin: 'dense',
+            size: 'small',
+        },
+        MuiFormHelperText: {
+            margin: 'dense',
+        },
+        MuiIconButton: {
+            size: 'small',
+        },
+        MuiInputBase: {
+            margin: 'dense',
+        },
+        MuiInputLabel: {
+            margin: 'dense',
+        },
+        MuiRadio: {
+            size: 'small',
+        },
+        MuiSwitch: {
+            size: 'small',
+        },
+        MuiTextField: {
+            margin: 'dense',
+            size: 'small',

```

Comment:
Just mu curiosity, are these prop defaults?

Reply:
Those can be removed. Thanks for asking!